src/EA.Weee.Web/Controllers/OrganisationRegistrationController.cs

```diff
@@ -289,6 +289,50 @@ public async Task<ActionResult> Type(OrganisationTypeViewModel model)
             return View(model);
         }
 
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<ActionResult> RepresentingCompanyDetails(RepresentingCompanyDetailsViewModel model)
+        {
+            if (!ModelState.IsValid)
+            {
+                var countries = await GetCountries();
+
+                model.Address.Countries = countries;
+
+                return View(model);
+            }
+
+            await transactionService.CaptureData(User.GetAccessToken(), model);
+
```

Comment:
@stevelee-civica Noticed that here we also need to call 
await transactionService.CompleteTransaction(User.GetAccessToken());
await cache.InvalidateOrganisationSearch();

As we need to mark the organisation transaction as complete. Could you add that after capture data and add the relevant unit tests similar to how others are done.